README.md

```diff
@@ -1,2 +1,85 @@
 # idb-views
 Vue component to produce different views for the https://www.cbd.int/biodiversity-day
+
+# dev
+
+Runs the code base locally in development. To run on of the different views change the view param in src/main.js.  index.html in the root is the entry file for the server.
+
+[ 'IdbMessages', 'IdbMessagesAdmin','IdbActions', 'IdbActionsAdmin', 'IdbMessagesCommunity', 'IdbActionsCountries' ]
+
+# preview
+
+Runs the code base after being built but not as a self embedded component.  Preview uses public/preview/index.html as the entry point template.  However, points directly to dist/dev/es/preview/index.html and uses dist/dev/es/ as the server root.
+
+Change the view param on the fly in dist/dev/es/preview/index.html
+
+# preview-widget 
+
+Runs the code base after being built and using the one line self embedding component. Preview-widget uses public/preview/widget/index.html as the entry point template. However, points directly to dist/dev/es/preview/widget/index.html and uses dist/dev/es/ as the server root.
+
+Change the view param on the fly in dist/dev/es/preview/widget/index.html
+
+# release-dev-s3
+1. build
+2. Replace reference to self in dist/import-map.json to s3 location.  '@scbd/idb-views': 'https://scbd-components.s3.amazonaws.com/%40scbd/idb-views%40${version}/dist/widget/index.js'
+3. In src/widget.js pass cdnUrl to WidgetBuilder
+
+Test the widget released on dev
+```
+<script 
+      type = "module"
+      src  = "https://scbd-components.s3.amazonaws.com/%40scbd/idb-views%400.0.2-alpha/dist/widget/index.js?d"
+      options = "{ 
+                    debug      : true,
+                    baseApiUrl : 'https://api.cbddev.xyz/api',
+                    accountsUrl: 'https://accounts.cbddev.xyz',
+                    year       : 2023,
+                    view       : 'IdbActionsAdmin',
+                    editUrl    : 'https://rjh.bioland.cbddev.xyz/idb-message'
+                  }"> 
```

Comment:
Year is required to reflect the current idb year.  Comms, copies and pasts all prev years in a drupal widget on the same page.  I can eliminate this and automate it all but they will need to input the previous years into the form.

One repo to show multiple views or multiple report for each view.  Was not sure which was best.

Since this is designed to be application independent, editUrl can change from application to application.  Presently message edit sites here: https://www.cbd.int/biodiversity-day/messages/community/submit
and action here: https://www.cbd.int/biodiversity-day/celebrations/submit.